war/src/main/scss/components/_dropdowns.scss

```diff
@@ -200,19 +200,9 @@ $dropdown-padding: 0.4rem;
       }
     }
 
-    &__badge {
-      position: relative;
+    .badge {
```

Comment:
This is not the right way to do it.
This should be `&__badge` (see comment for templates.js)